src/components/projects-summary/projects-summary.html

```diff
@@ -36,6 +29,7 @@ <h4 class="catalog-modal-title">
             <span class="fa fa-ellipsis-v"></span>
           </button>
           <ul class="uib-dropdown-menu dropdown-menu-right" aria-labelledby="dropdownKebab">
+            <li><a href="" ng-click="$ctrl.onViewMemebership(project)">View and Edit Membership</a></li>
```

Comment:
This is a long label :/ Maybe it should just be "View Membership"?

@jwforres any suggestions?